### PR TITLE
Add Alchemy

### PR DESCRIPTION
**Pull request by soodaayush**

### CODE REVIEW

File: entries/a/alchemy.com.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "Alchemy": {
+    "domain": "alchemy.com",
+    "contact": {
+      "twitter": "AlchemyPlatform"
+    },
+    "categories": [
+        "developer"
```
Comment (hkamran80) [inline entries/a/alchemy.com.json]:
```suggestion
      "developer"
```

File: entries/a/alchemy.com.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "Alchemy": {
+    "domain": "alchemy.com",
+    "contact": {
+      "twitter": "AlchemyPlatform"
```
Comment (hkamran80) [inline entries/a/alchemy.com.json]:
```suggestion
      "twitter": "AlchemyPlatform",
      "email": "support@alchemy.com"
```

File: img/a/alchemy.com.svg [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+<stop offset="0.770833" stop-color="#363FF9"/>
+<stop offset="1" stop-color="#5533FF"/>
+</radialGradient>
+<radialGradient id="paint2_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(90.0665) rotate(95.3323) scale(139.991 189.013)">
+<stop stop-color="#05D5FF"/>
+<stop offset="0.770833" stop-color="#363FF9"/>
+<stop offset="1" stop-color="#5533FF"/>
+</radialGradient>
+</defs>
+</svg>
```
Comment (hkamran80) [inline img/a/alchemy.com.svg]: Here's a minified SVG

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 170 145"><path fill="url(#a)" d="M141.4 92.8 87.6 1.5A3 3 0 0 0 85 0a3 3 0 0 0-2.6 1.5l-16 27.3a5.8 5.8 0 0 0 0 6l35 59.5a6 6 0 0 0 5.2 3h32.2a3 3 0 0 0 2.6-1.5 3 3 0 0 0 0-3z"/><path fill="url(#b)" d="m.4 140.6 53.8-91.4a3 3 0 0 1 2.6-1.4 3 3 0 0 1 2.6 1.4l16.1 27.4a5.8 5.8 0 0 1 0 5.9l-35 59.6a6 6 0 0 1-5.2 2.9H3a3 3 0 0 1-2.6-1.5 2.9 2.9 0 0 1 0-3Z"/><path fill="url(#c)" d="M59.4 145H167a3 3 0 0 0 2.6-1.5 2.9 2.9 0 0 0 0-3l-16.1-27.3a6 6 0 0 0-5.2-3H78a6.1 6.1 0 0 0-5.2 3l-16 27.4a2.9 2.9 0 0 0 1 4c.5.2 1 .4 1.5.4z"/><defs><radialGradient id="a" cx="0" cy="0" r="1" gradientTransform="matrix(-13 139 -188 -18 90 0)" gradientUnits="userSpaceOnUse"><stop stop-color="#05D5FF"/><stop offset=".8" stop-color="#363FF9"/><stop offset="1" stop-color="#53F"/></radialGradient><radialGradient id="b" cx="0" cy="0" r="1" gradientTransform="matrix(-13 139 -188 -18 90 0)" gradientUnits="userSpaceOnUse"><stop stop-color="#05D5FF"/><stop offset=".8" stop-color="#363FF9"/><stop offset="1" stop-color="#53F"/></radialGradient><radialGradient id="c" cx="0" cy="0" r="1" gradientTransform="matrix(-13 139 -188 -18 90 0)" gradientUnits="userSpaceOnUse"><stop stop-color="#05D5FF"/><stop offset=".8" stop-color="#363FF9"/><stop offset="1" stop-color="#53F"/></radialGradient></defs></svg>
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by soodaayush** [comment]
Changes made.

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**